Styles/navigation.css

```diff
@@ -0,0 +1,153 @@
+:root {
+    /* spark-ui css variables control */
+    /* secondary-button */
+    --secondary-button-focus-outline-color: #fff;
+    --secondary-button-bg-color: var(--secondary-color);
+    --secondary-button-color: #000;
+    --secondary-button-border: none;
+    --secondary-button-border-radius: 0.3em;
+    /* badge-on-button */
+    --button-badge-bg-color: transparent;
+    --button-badge-border: none;
+    --badge2-bg-color: #dd1313;
+    --badge2-border: none;
+}
+
+.logo {
+    width: 3.5rem;
+}
+
+.left-link {
+    text-decoration: none;
+    color: none;
+    padding-left: 0.5em;
+
+    /* flex-properties */
+    display: flex;
+    align-items: center;
+    gap: 1em;
+}
+
+.center-container {
+    width: max-content;
+    max-width: 30%;
+
+    /* flex-properties */
+    display: flex;
+    align-items: center;
+}
+
+.search-icon {
+    cursor: pointer;
+    background-color: #fff;
+    padding: 0.185em;
+    border: 1px solid #000;
+    border-radius: 0.3em 0 0 0.3em;
+    border-right: none;
+    font-size: 1.5rem !important;
+}
+
+.input {
+    font-size: 1rem;
+    padding: 0.45em 1em;
+    width: 30rem;
+    border: 1px solid #000;
+    border-radius: 0 0.3em 0.3em 0;
+    max-width: 100%;
+}
+
+.navigation-right-buttons-container {
+    align-items: center;
+}
+
+.navigation-button:focus {
+    border-radius: 0.2em;
+}
+
+.navigation-button-text {
+    padding-right: 0.5em;
+}
+
+.navigation .button.button-secondary {
+    font-size: 1.3rem;
+    padding: 0.4em 0.4em;
+    margin-left: 0.8em;
+}
+
+.button.button-secondary:hover {
+    background-color: var(--button-hover-color);
+}
+
+.badge-button {
+    font-size: 1.3rem;
+    background-color: transparent;
+    border: none;
+    border-radius: 0.2em;
+    padding: 0.3em;
+
+    /* flex-properties */
+    display: flex;
+    align-items: center;
+    gap: 0.5em;
+}
+
+.badge-button:hover {
+    background-color: #7dd3fc;
+    border-radius: 0.2em;
+}
+
+.badge-button:focus {
+    outline: var() solid 2px;
```

Comment:
i guess you forgot to add a color in var

Reply:
Thanks for pointing out